File: C/Iniciante/1072/1072.c
#include <stdio.h>
int main()
{
    int cont, n, valor, dentro = 0, fora = 0;
    scanf("%d", &n);
    for (cont = 1; cont <= n; cont++)
    {
        scanf("%d", &valor);
        if (valor >= 10 && valor <= 20)
            dentro++;
        else
            fora++;
    }
    printf("%d in\n", dentro);
    printf("%d out\n", fora);
    return 0;
}

File: Python/Iniciante/1072.py
# Problema 1072 - Beecrowd - Iniciante - Nível 1

# Leitura do valore de n
n = int(input())

# Contador para os números dentro do intervalo
#   Usando 'in_' pois 'in' é uma palavra reservada
in_ = 0

# Contador para os números fora do intervalo
out = 0

# Loop para cada valor
for i in range(n):
    # Leitura do valor
    x = int(input())

    # Verificação
    if 10 <= x <= 20:
        in_ += 1
    else:
        out += 1

# Saída apresentando os valores
print(f'{in_} in')
print(f'{out} out')